### DISCUSSION TITLE
what will replace `prisma deploy` in Prisma1

### DISCUSSION THREAD

**Discussion by alan345** [prisma/prisma#3216] [Q&A] [answered]
We are using Prisma1, without nexus. 

When we create a new object, 
1. we create it in `datamodel.graphql`
2. run `prisma deploy`. It updates the database and add data in the file `generated-schema.graphql`
3. we take what we need from `generated-schema.graphql` and add it into our `schema.grpahql` example:
```
type Mutation {
  updateAuthDevice(data: AuthDeviceUpdateInput!, where: AuthDeviceWhereUniqueInput!): AuthDevice
}

input AuthDeviceUpdateInput {
  validationToken: String
  lastLogin: DateTime
 ..
}

input AuthDeviceWhereUniqueInput {
  id: ID
}
```

Now with prisma2, we will not have `prisma deploy`. What is the good practice? Create every new objects by hand?

**Comment by ryands17** [comment, marked as answer]
Hey @alan345 👋 
With Prisma 2, you would need to specify the models that you need to expose via [Nexus](https://nexusjs.org/) and that will be easily available on a simple Node server that can be hosted.

Also if you directly want to generate CRUD operations, you can use [this](https://github.com/paljs/prisma-tools) package made by a community member.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
